app/models/tax_return.rb

```diff
@@ -23,4 +24,32 @@
 class TaxReturn < ApplicationRecord
   belongs_to :client
   belongs_to :assigned_user, class_name: "User", optional: true
+
+  INTAKE = "intake".freeze
+  PREP = "prep".freeze
+  REVIEW = "review".freeze
+  FINALIZE = "finalize".freeze
+  FILED = "filed".freeze
+  
+  STAGES = [INTAKE, PREP, REVIEW, FINALIZE, FILED].freeze
+
+  enum status: {
+    intake_in_progress: 101, intake_open: 102, intake_review: 103, intake_more_info: 104, intake_info_requested: 105, intake_needs_assignment: 106,
```

Comment:
I considered using the consts in order to make it more clear that these mapped to a stage -- 
```"#{INTAKE}_in_progress" => 101```

but I wasn't sure if it was useful/better/could have side effects?

Reply:
I like this the way it is pretty well, and I don't think `#{INTAKE}_` is clearer than `intake_`. I don't think there would be any side-effects from doing it the `#{INTAKE}_` way, though.

One alternative to consider is something like

```
STAGE_AND_STATUS = {
  intake: 
    {in_progress: 101, open: 102, review: 103, ...},
  prep:
    {ready_for_call: 201, more_info: 202, ...},
  ...
  }
```

Then you could compute something fancy with the `Hash[...]` constructor out of that.

Doesn't seem necessarily better, though.